Terraform.gitignore

```diff
@@ -35,3 +35,22 @@ override.tf.json
 # Ignore CLI configuration files
 .terraformrc
 terraform.rc
+
+# Ignore hcl files
+.terraform.lock.hcl
+*.terraform.*
```

Comment:
This is a bit dangerous
if for some reason I name a module `thorrsson.terraform.tf` I won't be able to commit it.

```suggestion
```